AlphaClicker/AHKManage.xaml.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+using Microsoft.Win32;
+
+
+namespace AlphaClicker
+{
+    /// <summary>
+    /// Interaction logic for AHKManage.xaml
+    /// </summary>
+    public partial class AHKManage : Window
+    {
+        public AHKManage()
+        {
+            InitializeComponent();
+        }
+
+        private void InstallButton_Click(object sender, RoutedEventArgs e)
+        {
+            string url = "https://www.autohotkey.com/download/ahk-v2.exe"; // Autohotkey download

```

Comment:
We want to keep AlphaClicker idendependent, not dependent on external software. If features from AHK is wished, implement them directly with the win32 api, not by proxing through ahk.